README.md

```diff
@@ -89,9 +89,11 @@ oneapi::mkl::blas::column_major::gemm(gpu_queue, transA, transB, m, ...);
 ```
 How to build an application with run-time dispatching:
 
+if OS is Linux, use icpx compiler. If OS is Windows, use icx compiler.
+Windows example:
 ```cmd
-$> dpcpp -fsycl –I$ONEMKL/include app.cpp
-$> dpcpp -fsycl app.o –L$ONEMKL/lib –lonemkl
+$> icx -fsycl –I$ONEMKL/include app.cpp
+$> icx -fsycl app.o –L$ONEMKL/lib –lonemkl
```

Comment:
This is linux example, could you please use icpx?

Reply:
Good catch! Thank you! Changed to icpx